articles/how-to-uninstall-fleetd.md

```diff
@@ -1,24 +1,53 @@
 # How to uninstall Fleet's agent (fleetd)
 
-How to remove fleetd from your device:
+You can uninstall fleetd directly on a device or remotely through Fleet.
 
-1. Download the uninstall script for [macOS](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/macos/scripts/uninstall-fleetd-macos.sh), [Windows](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/windows/scripts/uninstall-fleetd-windows.ps1), or [Linux](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/linux/scripts/uninstall-fleetd-linux.sh).
 
-2. Run the script.
-- To run the script on macOS, open the **Terminal** app.
-- Navigate to the script's directory by running this command: `cd /path/to/your/script`
-- Make the script executable: `chmod +x uninstall-fleetd-macos.sh`
-- Run it: `sudo ./uninstall-fleetd-macos.sh`
+## Uninstall fleetd on macOS
 
-How to uninstall fleetd from a host via Fleet (remotely):
+To remove fleetd from a Mac:
 
-1. Add the uninstall script for [macOS](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/macos/scripts/uninstall-fleetd-macos.sh), [Windows](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/windows/scripts/uninstall-fleetd-windows.ps1), or [Linux](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/linux/scripts/uninstall-fleetd-linux.sh) hosts to Fleet.
+1. Download the [macOS uninstall script](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/macos/scripts/uninstall-fleetd-macos.sh).
+2. Open the **Terminal** app.
+3. Navigate to where you saved the script: `cd /path/to/your/script`
+4. Make the script executable: `chmod +x uninstall-fleetd-macos.sh`
+5. Run the script: `sudo ./uninstall-fleetd-macos.sh`
 
-2. Head to the host's **Host details** page and select **Actions > Run script** to run the script.
 
-After performing these steps, the host will display as an offline host in the Fleet UI until you delete it.
+## Uninstall fleetd on Windows
 
-Are you having trouble uninstalling Fleetd on macOS, Windows, or Linux? Get help via one of our [support channels](https://fleetdm.com/support).
+To remove fleetd from a Windows device:
+
+1. Download the [Windows uninstall script](https://github.com/fleetdm/fleet/blob/main/it-and-security/lib/windows/scripts/uninstall-fleetd-windows.ps1).
+2. Open **PowerShell** as administrator (right-click and select **Run as administrator**).
+3. Navigate to where you saved the script: `cd C:\path\to\your\script`
+4. Run the script: `.\uninstall-fleetd-windows.ps1`
+
+Note: When running unsigned PowerShell scripts, you are likely to receive a warning, and will need to adjust the [Execution Policy](https://learn.microsoft.com/en-gb/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.5). One example is: `Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process`. This will bypass all warnings and prompts for the current PowerShell session. 
```

Comment:
```suggestion
> Note: When running unsigned PowerShell scripts, you are likely to receive a warning, and will need to adjust the [Execution Policy](https://learn.microsoft.com/en-gb/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.5). One example is: `Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process`. This will bypass all warnings and prompts for the current PowerShell session. 
```